src/frontend/efiling-frontend/src/components/input/Input.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from "react";
+import PropTypes from "prop-types";
+import "./Input.scss";
+
+export const Input = ({
+  input: {
+    label,
+    id,
+    value,
+    placeholder,
+    isRequired,
+    errorMsg,
+    styling,
+    note,
+    isReadOnly,
+    isControlled,
+  },
+  onChange,
+}) => {
+  let asterisk = "";
+
+  if (isRequired) {
+    asterisk = (
+      <span id="asterisk" className="bcgov-mandatory">
+        *
+      </span>
+    );
+  }
+
+  let labelPart = null;
+  const labelPartExists = label !== false;
+
+  if (label) {
+    labelPart = (
+      <label htmlFor={id} className="bcgov-label">
+        <div className="bcgov-input-label">{label}</div>
+        {asterisk}&nbsp;
```

Comment:
I always got my hand slapped for having nbsp's in PRs, but I forget if there is a suitable replacement for this. Ideas?

Reply:
I'm not entirely sure, I didn't realize they were an issue.

Reply:
Like I said, I was always told to do something else, but I forget what that something else was now.

Reply:
The only other alternative I can see suggested online is that you should CSS for this kind of spacing because it's more flexible than nbsp. This is one of the copy components from react shared components so it might be worth considering when we update those in the future.

Reply:
Can you make issues on the shared components repo for future work against this and the other thing we talked about yesterday?

Reply:
Sure, I will add that as well.